server/app.js

```diff
@@ -1,60 +1,92 @@
+const cors = require("cors");
+const express = require("express");
+const mongoose = require("mongoose");
+const bcrypt = require("bcrypt");
 
-const cors=require("cors")
-const express = require("express")
-const mongoose = require("mongoose")
-const bcrypt=require("bcrypt")
-
-const app=express();
+const app = express();
 
 app.use(cors());
-app.use(express.json())
+app.use(express.json());
 
 // mongoose.connect('mongodb+srv://akshithkumarkarla:9390733656a@queueup.qaqkgtb.mongodb.net/test')
 
-mongoose.set("strictQuery",false)
-const List = require('../server/UserSchema')
+mongoose.set("strictQuery", false);
+const List = require("../server/UserSchema");
 
-const appModal=require("./appModal");
-const UserSchema = require('./UserSchema')
+const appModal = require("./appModal");
+const UserSchema = require("./UserSchema");
 
-mongoose.connect("mongodb://localhost/DocsInfo",{
-    useNewUrlParser:true,
-    useUnifiedTopology:true,
-    family:4
-})
-.then(()=>{
-  app.listen(2917,()=>{
-    console.log("connected successfully")
+mongoose
+  .connect("mongodb://localhost/DocsInfo", {
```

Comment:
Make sure you know how to connect with atlas, later when you deploy you will have to use it.